security/utils.py

```diff
@@ -199,12 +199,12 @@ def serialize(self, qs):
         return data
 
     def backup_logs_and_clean_data(self, qs, backup_path):
-        for timestamp in qs.datetimes(self.get_timestamp_field(), 'day'):
+        for timestamp in qs.datetimes(self.get_timestamp_field(), 'day', tzinfo=utc):
```

Comment:
this is major bug. Wrong timezone info -> wrong db filter -> some logs was not removed